demosplan/DemosPlanCoreBundle/Logic/Import/Statement/AbstractStatementSpreadsheetImporter.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Import\Statement;
+
+use demosplan\DemosPlanCoreBundle\Entity\Statement\Statement;
+use demosplan\DemosPlanCoreBundle\Entity\Statement\Tag;
+use demosplan\DemosPlanCoreBundle\Exception\ClusterStatementCopyNotImplementedException;
+use demosplan\DemosPlanCoreBundle\Exception\CopyException;
+use demosplan\DemosPlanCoreBundle\Logic\Document\ElementsService;
+use demosplan\DemosPlanCoreBundle\Logic\Procedure\CurrentProcedureService;
+use demosplan\DemosPlanCoreBundle\Logic\Statement\StatementCopier;
+use demosplan\DemosPlanCoreBundle\Logic\Statement\StatementService;
+use demosplan\DemosPlanCoreBundle\Logic\User\CurrentUserService;
+use demosplan\DemosPlanCoreBundle\Logic\User\OrgaService;
+use PhpOffice\PhpSpreadsheet\IOFactory;
+use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
+use Symfony\Component\Finder\SplFileInfo;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Webmozart\Assert\Assert;
+
+abstract class AbstractStatementSpreadsheetImporter implements StatementSpreadsheetImporterInterface
+{
+    protected readonly NotBlank $notNullConstraint;
+
+    /**
+     * @var list<Statement>
+     */
+    protected array $generatedStatements = [];
+
+    /**
+     * {@link Tag} entities that do not yet exist in the database and were created during the import.
+     *
+     * @var list<Tag>
+     */
+    protected array $generatedTags = [];
```

Comment:
Are the statements also not yet existing in the database and were created during import? Can we adjust the docblock of $generatedStatements and by bring it into line with the one of generatedTags?